README.md

```diff
@@ -35,7 +35,7 @@ Below is a callback function for validating a dl field using CodeIgniter's `form
         'ID'=>"(^[a-zA-Z]{2}[0-9]{6}[a-zA-Z]{1}$)|(^[0-9]{9}$)", //2Alpha+6Numeric+1Alpha or 9Numeric
         'IL'=>"^[a-zA-Z]{1}[0-9]{11,12}$", //1Alpha+11-12Numeric
         'IN'=>"(^[a-zA-Z]{1}[0-9]{9}$)|(^[0-9]{9,10}$)", //1Alpha+9Numeric or 9-10Numeric
-        'IA'=>"^[0-9]{9}|([0-9]{3}[a-zA-Z]{2}[0-9]{4}$)", //9Numeric or 3Numeric+2Alpha+4Numeric
+        'IA'=>"(^[0-9]{9}$)|([0-9]{3}[a-zA-Z]{2}[0-9]{4}$)", //9Numeric or 3Numeric+2Alpha+4Numeric
```

Comment:
I had someone email me about this one specifically. I changed it to be 
`^[0-9]{9}|([0-9]{3}[a-zA-Z]{2}[0-9]{4})$`